dist/.magento.env.yaml

```diff
@@ -457,6 +457,18 @@
 #                default_setup:                                                                                       #
 #                  connection: default                                                                                #
 #######################################################################################################################
+# ERROR_REPORT_DIR_NESTING_LEVEL - Subdirectories nesting level, which indicates how many nested subdirectories will  #
```

Comment:
Add description that this variable is ignored if you have some file in git

Reply:
Done